index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE <!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>ui-challenge-1</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="styles/normalize.css" />
+  <link rel="stylesheet" href="styles/normalize.css" />
+  <link rel="stylesheet" href="styles/base.css" />
+  <link rel="stylesheet" href="styles/layout.css" />
+  <link rel="stylesheet" href="styles/modules.css" />
+</head>
+
+<body>
+  <section id="mobile">
```

Comment:
Having two entirely separate versions of your site for desktop and mobile is a solution that works, but it leads to a lot of bugs. In this case, you have different content on the two versions of your site (the desktop version is missing the letters), and those letters show up in the wrong places on mobile. A more correct solution to this problem would use the same elements on both versions of the site, and would change their sizes or positioning with media queries, rather than showing and hiding huge portions of the site.